retweets.go

```diff
@@ -75,7 +75,7 @@ func postRetweet(ctx context.Context, c *client, uid string, tid string) (*PostR
 	if err != nil {
 		return nil, fmt.Errorf("post retweet new request with ctx: %w", err)
 	}
-	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.bearerToken))
+	req.Header.Set("Authorization", fmt.Sprintf("OAuth %s", c.bearerToken))
```

Comment:
I've mixed up the correction of a mistake in a previous retweet pr.
F.Y.I https://developer.twitter.com/en/docs/twitter-api/tweets/retweets/api-reference/post-users-id-retweets